nevergrad/optimization/optimizerlib.py

```diff
@@ -990,18 +990,21 @@ def __init__(self, instrumentation: Union[int, Instrumentation], budget: Optiona
     @property
     def bo(self) -> BayesianOptimization:
         if self._bo is None:
+            params = self._parameters
             bounds = {f'x{i}': (0., 1.) for i in range(self.dimension)}
             seed = np.random.randint(2**32, dtype=np.uint32)
             self._bo = BayesianOptimization(self._fake_function, bounds, random_state=np.random.RandomState(seed))
+            if self._parameters.gp_parameters is not None:
+                self._bo.set_gp_parameters(**self._parameters.gp_parameters)
```

Comment:
this actually should be `self._bo.set_gp_params(**self._parameters.gp_parameters)`

Reply:
Thanks, I was a bit too fast with it sorry.
Btw, don't hesitate to create a new issue (or PR) for this kind of comment, I nearly did not see it in my emails :s

Reply:
> Thanks, I was a bit too fast with it sorry. It should now be solved
> Btw, don't hesitate to create a new issue (or PR) for this kind of comment, I nearly did not see it in my emails :s

I really don't feel like making a new PR for just correcting a very small typo :D